doc/nrf/releases_and_maturity/migration/migration_guide_2.6.rst

```diff
@@ -60,6 +79,547 @@ Recommended changes
 
 The following changes are recommended for your application to work optimally after the migration.
 
+* For the Matter samples and applications
+
+  * The new API and helper modules have been added to the :file:`samples/matter/common` directory.
+    All Matter samples and applications have been changed to use the common software modules.
+
+    You can follow the new approach and migrate your application to use the common software modules.
+    This will signicantly reduce the size of a code required to be implemented in the application.
+    You can also choose to keep using the previous approach, but due to the structural differences, it may be harder to use Matter samples and applications as a reference for an application using the older approach.
+
+    In the following steps use the :ref:`matter_template_sample` sample as an example.
+    To migrate the application from |NCS| v2.5.0 and start using common software modules from |NCS| v2.6.0:
+
+    * Replace the code used for initialization and handling of the board's components, like LEDs or buttons, with the common ``board`` module.
+      The ``board`` module handles buttons and LEDs in a way consistent with Matter samples UI.
+      It uses the ``task_executor`` common module for posting a board related events.
+      You can also use the ``task_executor`` module for posting and dispatching events in your application.
+
+      To replace the |NCS| v2.5.0 compliant implementation with the ``board`` module, complete the following steps:
+
+      * Remove the following code from the :file:`app_task.h` file:
```

Comment:
```suggestion
      1. Remove the following code from the :file:`app_task.h` file:
```
I think this level is not necessary to do in order, but using a numbered list helps readability since there are so many levels bullets otherwise.

You'll need to indent all `code-block`s by one space.

Reply:
Ok, applied.